applications/StructuralMechanicsApplication/custom_strategies/custom_schemes/explicit_central_differences_scheme.hpp

```
@@ -709,9 +716,34 @@ class ExplicitCentralDifferencesScheme
         KRATOS_CATCH("")
     }
 
-    ///@}
-    ///@name Operations
-    ///@{
+    /**
+     * @brief This method provides the defaults parameters to avoid conflicts between the different constructors
+     * @return The default parameters
+     */
+    Parameters GetDefaultParameters() const override
+    {
+        Parameters default_parameters = Parameters(R"(
+        {
+            "name"                       : "central_differences",
```

where did u take these defaults from?

---

in particular i would set by default the max_delta_time to a negative number. if after the validate it remains negative, than i would throw an error (users need to set a realistic max!!)